src/components/track/desktop/TrackTile.tsx

```diff
@@ -93,6 +96,42 @@ const TrackTile = memo(
       (e: MouseEvent) => e.stopPropagation(),
       []
     )
+    const hideShare: boolean = fieldVisibility
+      ? fieldVisibility.share === false
+      : false
+    const hidePlays = fieldVisibility
+      ? fieldVisibility.play_count === false
+      : false
+
+    const shareButtonContainer = (
```

Comment:
let's pull this into a function like so 
https://github.dev/AudiusProject/audius-client/blob/637c9b4f337c117ee54ecead959693e77990e825/src/containers/download-buttons/DownloadButtons.tsx#L89